crypto/s2n_hmac.c

```diff
@@ -44,41 +62,37 @@ int s2n_hmac_hash_alg(s2n_hmac_algorithm hmac_alg, s2n_hash_algorithm *out)
     default:
         S2N_ERROR(S2N_ERR_HMAC_INVALID_ALGORITHM);
     }
-    return 0;
+    return S2N_SUCCESS;
 }
 
 int s2n_hmac_digest_size(s2n_hmac_algorithm hmac_alg, uint8_t *out)
 {
     s2n_hash_algorithm hash_alg;
     GUARD(s2n_hmac_hash_alg(hmac_alg, &hash_alg));
     GUARD(s2n_hash_digest_size(hash_alg, out));
-    return 0;
+    return S2N_SUCCESS;
 }
 
 /* Return 1 if hmac algorithm is available, 0 otherwise. */
```

Comment:
We should remove this comment now, right?

Reply:
Good catch!!

Reply:
Removed it.